test/test_library.py

```diff
@@ -462,17 +452,6 @@ def test_destination_with_replacements(self):
         self.i.album = "bar"
         assert self.i.destination() == np("base/ber/foo")
 
-    def test_destination_with_replacements_argument(self):
-        self.lib.directory = b"base"
-        self.lib.replacements = [(re.compile(r"a"), "f")]
-        self.lib.path_formats = [("default", "$album/$title")]
-        self.i.title = "foo"
-        self.i.album = "bar"
-        replacements = [(re.compile(r"a"), "e")]
-        assert self.i.destination(replacements=replacements) == np(
-            "base/ber/foo"
-        )
-
```

Comment:
Maybe, we want to keep this test? I read this more of a test about `destination()` honoring the `replacements` argument, rather than testing the replacement mechanism in detail.

Reply:
`destination()` does not have such parameter anymore, see 366a5fd

Reply:
Sorry, must've missed that!